pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -rsxX --reuse-db --tb=native
+testpaths = kuma
+python_files = test*.py
```

Comment:
Ugh. Github keeps hiding things on me.

There are a handful of files named `tests.py`. So we can change this, but we'd have to rename files.